docs/usage/configuration-options.md

```diff
@@ -2659,15 +2659,21 @@ Avoid setting `rebaseWhen=never` and then also setting `prCreation=not-pending`
 
 ## recreateClosed
 
-By default, Renovate will detect if it has proposed an update to a project before and not propose the same one again.
-For example the Webpack 3.x case described above.
-This field lets you customize this behavior down to a per-package level.
-For example we override it to `true` in the following cases where branch names and PR titles need to be reused:
+By default, Renovate detects if it proposed an update to a project before, and not propose the same one again.
+For example the Webpack 3.x case described in the `separateMajorMinor` documentation.
+You can use `recreateClosed` to customize this behavior down to a per-package level.
+For example we override it to `always` in the following cases where branch names and PR titles need to be reused:
 
 - Package groups
 - When pinning versions
 - Lock file maintenance
 
+There are total three possible options:
```

Comment:
```suggestion
You can select which behavior you want from Renovate bot:
```

How about this? Saying "There are total three possible options" is vague. Maybe a sentence like this one is clearer?

Reply:
Yes, I feel so too. Thanks♥